delft/utilities/Utilities.py

```diff
@@ -15,6 +16,7 @@
 
 import argparse
 import truecase
+from keras import backend as K
```

Comment:
This is not valid with tf2, usually to be replaced by:

```python
from tensorflow.keras import backend as K
```

or depending on what is used 

```python
from tensorflow,compat,v1,keras import backend as K
```

But the best is however to update the call to something only tf2, so that we don't use the backend stuff.

Reply:
Looking at it, we can simply move the `dot_product()` to `Attention.py` and normally use what is already imported there.

Reply:
OK